src/main/kotlin/no/nav/syfo/infrastructure/kafka/VurderingProducer.kt

```diff
@@ -0,0 +1,78 @@
+package no.nav.syfo.infrastructure.kafka
+
+import no.nav.syfo.application.IVurderingProducer
+import no.nav.syfo.domain.DocumentComponent
+import no.nav.syfo.domain.JournalpostId
+import no.nav.syfo.domain.ManglendeMedvirkningVurdering
+import no.nav.syfo.domain.Personident
+import no.nav.syfo.domain.Varsel
+import no.nav.syfo.domain.Veilederident
+import no.nav.syfo.domain.VurderingType
+import no.nav.syfo.util.configuredJacksonMapper
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.apache.kafka.common.serialization.Serializer
+import org.slf4j.LoggerFactory
+import java.time.OffsetDateTime
+import java.util.UUID
+
+class VurderingProducer(private val producer: KafkaProducer<String, VurderingRecord>) : IVurderingProducer {
+
+    override fun publishVurdering(vurdering: ManglendeMedvirkningVurdering): Result<ManglendeMedvirkningVurdering> =
+        try {
+            producer.send(
+                ProducerRecord(
+                    TOPIC,
+                    vurdering.personident.asProducerRecordKey(),
+                    VurderingRecord.fromVurdering(vurdering),
+                )
+            )?.also { it.get() }
+            Result.success(vurdering)
+        } catch (e: Exception) {
+            log.error("Exception was thrown when attempting to send vurdering: ${e.message}")
+            Result.failure(e)
+        }
+
+    companion object {
+        private const val TOPIC = "teamsykefravr.manglende-medvirkning-vurdering"
+        private val log = LoggerFactory.getLogger(VurderingProducer::class.java)
+    }
+}
+
+fun Personident.asProducerRecordKey(): String = UUID.nameUUIDFromBytes(value.toByteArray()).toString()
+
+data class VurderingRecord(
+    val uuid: UUID,
+    val personident: Personident,
+    val veilederident: Veilederident,
+    val createdAt: OffsetDateTime,
+    val begrunnelse: String,
+    val document: List<DocumentComponent>,
```

Comment:
Trenger kanskje ikke ta med `document` her heller? Usikker på om vi har pleid å ta med det andre steder :thinking: Kan ikke helt se for meg at noen av våre apper trenger konsumere det og formatet der egner seg heller ikke så godt til telling hos DVH.

Reply:
Ja, vi ser på konsumeringen nå, og vi tar ikke inn document andre steder. Så støtter at vi ikke trenger å sende dette 👍🏼

Reply:
Bra sett! 🚀